BSchoolland/v2v-chatbot-api
# feat: add tool usage indicators in chatbot responses

Improve tool usage indicators in chatbot responses

Enhances the visual appearance and behavior of tool usage indicators in the chatbot interface.

Changes:
- Added proper text wrapping for long URLs
- Improved spacing and layout
- Only show successful tool calls
- Show full URLs in indicators
- Fixed stacking behavior

Testing:
1. Start the server and open the chatbot interface
2. Send a message that requires tool usage (e.g., asking about website content)
3. Verify that tool usage indicators appear with full URLs
4. Confirm that long URLs wrap properly without scrollbars
5. Verify that failed tool calls don't show indicators

Fixes #58
